pyanaconda/modules/storage/nvdimm/nvdimm.py

```diff
@@ -101,6 +101,9 @@ def get_devices_to_ignore(self):
         devices_to_ignore = set()
 
         for ns_name, ns_info in nvdimm.namespaces.items():
+            if ns_info is None and ns_info.blockdev is None:
```

Comment:
The `ns_info` variable shouldn't be `None`.

Reply:
Please, remove `if ns_info.blockdev:` below.

Reply:
Please, extend the unit tests that call this method. I think it should be enough to define a namespace without a block device in `_check_ignored`.